SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -59,7 +59,8 @@ bool CreateSpawnerEntity(Worker_SnapshotOutputStream* OutputStream)
 	SpawnerEntity.component_count = Components.Num();
 	SpawnerEntity.components = Components.GetData();
 
-	return Worker_SnapshotOutputStream_WriteEntity(OutputStream, &SpawnerEntity) != 0;
+	Worker_SnapshotOutputStream_WriteEntity(OutputStream, &SpawnerEntity);
+	return Worker_SnapshotOutputStream_GetState(OutputStream).stream_state == WORKER_STREAM_STATE_GOOD;
```

Comment:
So I am actually not sure how this previously worked. Did 0 indicate success or failure? I return true on a good state, and false on any other, which seems to work.

Reply:
0 is implicitly converted to false. Everything else is true.